code/modules/surgery/surgery.dm

```diff
@@ -115,6 +115,8 @@
 		if(implement_type)	//this means it isn't a require nd or any item step.
 			prob_chance = allowed_tools[implement_type]
 		prob_chance *= get_location_modifier(target)
+		prob_chance *= get_pain_modifier(target)//operating on conscious people is hard.
```

Comment:
What if the step does not cause pain, like repairing/attaching a bionic limb?

Reply:
Good question, I hadn't considered that. Though attaching a bionic limb probably should be painful, since I assume you'd need to attach the nerves to the new limb and so on.

I agree on the repairing, though. Will look into this.